src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingBarracks.java

```diff
@@ -110,6 +111,30 @@ public void onUpgradeComplete(final int newLevel)
                     ((BuildingBarracksTower) building).addBarracks(getLocation());
                 }
             }
+
+            //TODO: Implement the following properly. Once we've got textures and etc for a Substitution block.
+            /*for (final Tuple<BlockPos, EnumFacing> tower : getBarracksTowers())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)